pipenv/core.py

```diff
@@ -1634,26 +1768,6 @@ def fallback_which(command, location=None, allow_global=False, system=False):
     return ""
 
 
-def which_pip(project):
```

Comment:
We stopped using this method in the prior effort to only reference the internal vendor'd `pip`.